firecracker-control/cmd/containerd/main.go

```diff
@@ -46,6 +46,10 @@ import (
 	_ "github.com/containerd/containerd/runtime/v2"
 	_ "github.com/containerd/containerd/runtime/v2/runc/options"
 
+	// Snapshotters
+	_ "github.com/containerd/containerd/snapshots/devmapper"
```

Comment:
Do we want this one or our out-of-process gRPC proxy plugin?

Reply:
This still allows you to use out-of-process plugins as needed. But I think we should start moving towards embedded version as it's more up to date and 1.3 release is very close.